### PR TITLE
recovery of the index, connected shop page which pops and creation of the shop page with carousel

### PR DESCRIPTION
**Pull request by 100PBaguuette**
- recovery of the index only 3 card displayed, text placed below with a button.
image

- once the user is connected, the index page displays a new personalized code in order to have a shop, in the center the search bar, on the left a carousel with several cards. On the right the map of the Google API which covers it.
image

### CODE REVIEW

File: app/views/bicycles/index.html.erb [hunk 1]
```diff
@@ -1,54 +1,150 @@
...
+          </div>
+        </form>
+        <button type="submit" class="btn btn-primary mx-6">Rechercher</button>
+      </div>
+      <div id="bicycles" class="row d-flex justify-content-center">
+        <div id="carouselExampleControls" class="carousel slide col-md-7 mx-auto" data-bs-ride="carousel">
+          <div class="carousel-inner">
+            <% @bicycles.each_slice(4).with_index do |group, index| %>
+              <div class="carousel-item <%= index == 0 ? 'active' : '' %>">
+                <div class="row">
```
Comment (masechkacat) [inline app/views/bicycles/index.html.erb:28]: `<div class="row justify-content-center">`

Comment (ikramiste) [reply app/views/bicycles/index.html.erb:28]: @100PBaguuette Do you wish to proceed with this edit ? 💯 
Ready for approving those changes once done for a test in production asap. 👍

**Comment by 100PBaguuette** [comment]
Carousel Improvement:
margin removed, addition of two images for the right and left arrows of the slide, for greater readability.
image


Edit:
Color of prices/times on all cards.
image


Responsive shop page + resumption of the map on the mobile version.
image

Checking links and updating for “welcome” and “shop” pages.

**Comment by ikramiste** [comment]
fixed seed conflict - your code was not altered 👍

**Pull request merged**